ocpmodels/models/schnet_core.py

```diff
@@ -0,0 +1,260 @@
+# Pytorch geometric added this in v1.5, while we're currently on v1.4.
```

Comment:
Any reason we aren't on 1.5 now? I've been on 1.5 since I've started and had no problems.

Reply:
Good to know that it's been working fine! Yeah, we can move to 1.5 soon. I'll check again to make sure the upgrade doesn't break anything and update the setup instructions.